packages/golden_toolkit/lib/src/device.dart

```diff
@@ -0,0 +1,61 @@
+import 'dart:ui';
```

Comment:
extracted out to separate file. I'm planning on updating pumpWidgetBuilder to use this instead of more granular size/textscale arguments.